wrangler-service/src/main/java/io/cdap/wrangler/service/directive/WorkspaceHandler.java

```diff
@@ -620,7 +620,7 @@ private <E extends Exception> List<Row> executeRemotely(String namespace, List<S
     RemoteDirectiveRequest directiveRequest = new RemoteDirectiveRequest(recipe, systemDirectives,
                                                                          namespace, detail.getSampleAsBytes());
     RunnableTaskRequest runnableTaskRequest = RunnableTaskRequest.getBuilder(RemoteExecutionTask.class.getName())
-      .withParam(GSON.toJson(directiveRequest))
+      .withParam(GSON.toJson(directiveRequest)).withNamespace(namespace)
```

Comment:
nit: You can put the new filed in a new line to keep it consistent with the surrounding code.

Reply:
ack, done.